requirements.txt

```diff
@@ -1,5 +1,5 @@
 #
-# This file is autogenerated by pip-compile with python 3.6
+# This file is autogenerated by pip-compile with python 3.9
```

Comment:
Don't know if the dependency bumps will be an issue since you didn't use a Python 3.6 environment to generate the dependency matrix - did you test this?

Reply:
I did test this but forgot to re-do the pip-compile with 3.6, lemme fix it